beacon-chain/core/blocks/attestation.go

```diff
@@ -107,22 +107,53 @@ func VerifyAttestationNoVerifySignature(
 		return err
 	}
 	c := helpers.SlotCommitteeCount(activeValidatorCount)
-	if uint64(att.GetData().CommitteeIndex) >= c {
-		return fmt.Errorf("committee index %d >= committee count %d", att.GetData().CommitteeIndex, c)
-	}
 
-	if err := helpers.VerifyAttestationBitfieldLengths(ctx, beaconState, att); err != nil {
-		return errors.Wrap(err, "could not verify attestation bitfields")
-	}
+	var indexedAtt ethpb.IndexedAtt
 
-	// Verify attesting indices are correct.
-	committee, err := helpers.BeaconCommitteeFromState(ctx, beaconState, att.GetData().Slot, att.GetData().CommitteeIndex)
-	if err != nil {
-		return err
-	}
-	indexedAtt, err := attestation.ConvertToIndexed(ctx, att, committee)
-	if err != nil {
-		return err
+	if att.Version() < version.Electra {
```

Comment:
Same feedback as my other comment, could you refactor the old logic to its own method and put the old spec definition there?

Reply:
It's not as straightforward here because the other function was basically replaced in whole. Here only a small subset of validation in the middle of the function is different.